maquetado/setup/scss/titles.scss

```diff
@@ -0,0 +1,20 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title {
+  display: flex;
+  position: relative;
+
+  .title-decoration{
+    height: $title-decoration-height;
+    width: $title-decoration-width;
+  }
+
+  .title-text {
+    align-self: center;
+    font-family: 'Roboto', sans-serif;
+    font-size: $text-big;
+    margin: 5px 52px;//too specific?
+    position: absolute;
+  }
```

Comment:
Same, you could use `h1` as title instead and use `position: relative` as an anchor. Then, use a pseudo element to add the decorative image. That way you can avoid using flex here and the div tag